plugins/input/pgraster/pgraster_datasource.cpp

```diff
@@ -878,8 +878,9 @@ featureset_ptr pgraster_datasource::features_with_context(query const& q,process
         std::string col = geometryColumn_;
 
         if ( use_overviews_ ) {
-          std::string sch = schema_;
-          std::string tab = mapnik::sql_utils::unquote_double(raster_table_);
+          std::string sch = overviews_[0].schema;
```

Comment:
`use_overviews_` might be true but you might not have overviews.

`use_overviews_` is a configuration parameter to determine if overview tables must be used or not. We at least should check `overviews_[0]` is defined.